js/engine/tests/__snapshots__/index.test.js.snap

```diff
@@ -86,7 +86,7 @@ exports[`yaml parser interpolates metavariables in rule message 1`] = `
           },
           "extra": {
             "engine_kind": "OSS",
-            "message": "Print content is $A",
+            "message": "Print content is "Hello World"",
```

Comment:
hmm does this work? the double quotes should be escaped no?

Reply:
it's not ok, this is not valid json, so whatever generate those, if it's us, it's not good.

Reply:
The one thing I don't understand is, as far as I can see, this is generated directly by the Jest library, generating the JSON from actual Javascript objects. This wasn't produced by us. So I can't see why this would have been our fault (it seems like it would be an issue with Jest if anything)

Reply:
Seems this is intended behavior from Jest: https://github.com/jestjs/jest/issues/13776

We could change this, but I don't care altogether too much if it isn't our fault.

Reply:
ah ok. thanks for looking. We're good to go then.